src/Pie.js

```diff
@@ -18,52 +37,94 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       r,
       startAngle,
       startAngle + arcAngle,
-    );
-  } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
```

Comment:
that's interesting, so the old logic is useless, it's identical with the web version? That's cool

Reply:
so should I go ahead and remove the Platform.OS !== 'web' I am unsure how to test for web

Reply:
How would this ever work in web? We're dependent on `react-native-community/art`